### PR TITLE
connectedhomeip: initial integration

### PR DESCRIPTION
**Pull request by abcSup**
This PR is to bring existing fuzzers in the connectedhomeip project to OSS-Fuzz.

CC: @gmarcosb, @andy31415

### CODE REVIEW

File: projects/connectedhomeip/project.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+auto_ccs:
+  - "mboyington@google.com"
+sanitizers:
+  - address
+  - undefined
+  - memory
+fuzzing_engines:
+  - libfuzzer
+  - honggfuzz
+  - centipede
```
Comment (jonathanmetzman) [inline projects/connectedhomeip/project.yaml:13]: no afl?

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: projects/connectedhomeip/project.yaml (hunk 1 above)
Comment (abcSup) [reply projects/connectedhomeip/project.yaml:13]: I had it working locally, but I would need to submit another PR to the target project before enabling it here.

**Comment by abcSup** [comment]
Thank you!

**Pull request merged**